spec/jobs/send_daily_alert_email_job_spec.rb

```diff
@@ -71,10 +71,8 @@
       job.vacancies_for_subscription(subscription)
     end
 
-    context 'with algolia search', algolia: true do
+    context 'subscription created before algolia' do
```

Comment:
Lines 74-137 inclusive (on the original file) need to go I think

Reply:
I left those in because I wasn't sure how many (any?) old-school subscriptions we might still have running.  If they are all gone, I'm happy to zap those lines.

Reply:
I just went ahead and killed them.